src/api.v2/helpers/pipeline/gem2s.js

```diff
@@ -0,0 +1,177 @@
+const _ = require('lodash');
+const AWSXRay = require('aws-xray-sdk');
+
+// const constants = require('../general-services/pipeline-manage/constants');
+const constants = require('./constants');
+// const getPipelineStatus = require('../general-services/pipeline-status');
+const getPipelineStatus = require('./getPipelineStatus');
+// const { createGem2SPipeline } = require('../general-services/pipeline-manage');
+const { createGem2SPipeline } = require('./pipelineConstruct');
+
+const Sample = require('../../model/Sample');
+const Experiment = require('../../model/Experiment');
+const ExperimentExecution = require('../../model/ExperimentExecution');
+
+const sendNotification = require('./hooks/sendNotification');
+const HookRunner = require('./hooks/HookRunner');
+
+const validateRequest = require('../../../utils/schema-validator');
+const getLogger = require('../../../utils/getLogger');
+
+const logger = getLogger();
+
+const hookRunner = new HookRunner();
+
+const saveProcessingConfigFromGem2s = ({ experimentId, item }) => {
+  new Experiment().updateById(experimentId, { processing_config: item });
+};
+
+const continueToQC = () => {
+  // const continueToQC = (payload) => {
+  // const { experimentId } = payload;
+  // we need to change this once we rework the pipeline message response
+  // const authJWT = payload.authJWT || payload.input.authJWT;
+  // await createQCPipeline(experimentId, [], authJWT);
+};
```

Comment:
These hooks are small and only used here, should I still split them up into different files?